src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/entity/VideoBooking.kt

```diff
@@ -41,15 +47,58 @@ class VideoBooking private constructor(
 
   val createdBy: String,
 
-  val createdTime: LocalDateTime = LocalDateTime.now(),
+  val createdTime: LocalDateTime = now(),
 ) {
 
+  @OneToMany(mappedBy = "videoBooking", fetch = FetchType.LAZY, cascade = [CascadeType.ALL], orphanRemoval = true)
+  private val prisonAppointments: MutableList<PrisonAppointment> = mutableListOf()
+
+  @Enumerated(EnumType.STRING)
+  var statusCode: StatusCode = StatusCode.ACTIVE
+    private set
+
   var amendedBy: String? = null
 
   var amendedTime: LocalDateTime? = null
 
   fun isCourtBooking() = bookingType == "COURT"
 
+  fun addAppointment(
```

Comment:
I have kept this fairly basic in terms of validation (none at the mo) at the moment but we can enrich if we feel the need e.g. prevent addition of duplicate appointments, appointments must be in future etc.